.github/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+## Scope and purpose
+
+Fixes #<!-- ISSUE-ID -->. Dependent on #<!-- PULL-REQUEST-ID -->. <!-- Reference: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue -->
+
+Add a description of what this PR does and why it is needed. If the linked ticket(s) fully
+cover it, you can omit this.
+
+### This pull request <!-- remove things that do not apply -->
+* adds/changes/removes a dependency
+* changes the API
+
+
+## Contributor Checklist
+
+Every pull request should have this checklist filled out, no matter how small it is.
+More information about contributing to Zino can be found in the
+[README](https://github.com/Uninett/zino/blob/master/README.md#developing-zino).
+
+<!-- Add an "X" inside the brackets to confirm -->
+<!-- If not checking one or more of the boxes, please explain why below each. -->
+
+* [ ] Added a changelog fragment for [towncrier](https://github.com/Uninett/zino/blob/master/README.md#before-merging-a-pull-request)
+* [ ] Added/amended tests for new/changed code <!-- In case CodeCov Upload makes the tests fail, simply rerun them a few minutes later -->
+* [ ] Added/changed documentation
+* [ ] Linted/formatted the code with black, ruff and isort, easiest by using [pre-commit](https://github.com/Uninett/zino/blob/master/README.md#code-style)
```

Comment:
Here if would also like to add a check for how the commit messages are formulated. Does anyone have a good link to which way we prefer?

Reply:
Good question.  I mean, I know which way I prefer, but I don't know where to find someone else's writing about it.  Should we just write our own policy?  Comments, @hmpf ?

Reply:
I am writing one for argus, see Uninett/Argus#828.

My preferences, in short:

* First line:
  * Start with capital letter
  * Do not end with punctuation
  * Fifty characters or less
  * Fill out the pattern "If applied, this commit will .." aka. *verb in the infinitive*
  * Don't waste space on issue numbers and  prefixes AND NO EMOJI
* Second line: blank if needed
* Body:
  * Starts on the third line
  * Wraps at 79, preferably 72 characters
  * May contain issue numbers, links etc.

Web-searching a little, programmers still do not understand what imperative means... For instance https://cbea.ms/git-commit/: They use (as I use) the pattern "If applied, this commit will COMMIT MESSAGE". The verb in that COMMIT MESSAGE is in the *infinitive*, not the *imperative*. *sigh*

The difference is obvious as soon as you need a negative verb:

* This commit will "Prevent a race condition" -> infinitive
* This commit will "Do not do x and y at the same time" -> imperative

Reply:
But yeah, we could just link to https://cbea.ms/git-commit/

Reply:
We could have the check as:

* [ ] The first line of the commit message continues the sentence "If applied, this commit will", starts with a capital letter, does not end with punctuation and is 50 characters or less long. See https://cbea.ms/git-commit/